app/bundles/CampaignBundle/Entity/LeadRepository.php

```diff
@@ -383,10 +383,12 @@ public function getCountsForCampaignContactsBySegment($campaignId, ContactLimite
     }
 
     /**
-     * @param      $campaignId
-     * @param bool $campaignCanBeRestarted
```

Comment:
You'll have to revert these lines back. And please add an `int` type for `$campaignId`